_data/events.yml

```diff
@@ -1347,6 +1347,23 @@
   twitter: "https://twitter.com/itenium_be/status/1574320233984114688"
   githubUrl: "https://github.com/itenium-be/UnitTesting"
 
+
+- name: "CO2duino"
+  subTitle: "CO2 Meter"
+  date: 2022-06-29
+  track: "TechEvent"
+  summary: >
+       Monitored and managed your indoor air quality with 
+       precision using the CO2 Meter—your key to a healthier, 
+       more comfortable environment.
+  level: Introduction
+  content: "Theoretical"
+  type: ["Virtual"]
+  duration: "1h"
+  image: "assets/images-events/CO2duino-1.jpeg"
```

Comment:
I think there are probably like a gazillion images for this one as well
Can you check?